runtime/compiler/runtime/JITServerAOTCache.hpp

```diff
@@ -30,7 +30,9 @@
 #include "runtime/JITServerAOTSerializationRecords.hpp"
 
 namespace TR { class Monitor; }
+namespace J9 { class J9SegmentProvider;}
 
+#define LOAD_AOT_CACHE_REQUEST (JITServer::ServerStream *)0x1
```

Comment:
Can we name this `LOAD_AOTCACHE_REQUEST` to match `AOTCache` used in the rest of the code and make it more grep-friendly?

Reply:
Done